fe/fe-core/src/main/java/org/apache/doris/catalog/StorageVaultMgr.java

```diff
@@ -156,6 +161,56 @@ private Cloud.StorageVaultPB.Builder buildAlterS3VaultRequest(Map<String, String
         return alterObjVaultBuilder;
     }
 
+    private static Cloud.ObjectStoreInfoPB.Builder getObjStoreInfoPB(Map<String, String> properties) {
```

Comment:
I think this method should be moved to S3Properties?
1. properties -> S3StorageProperties
2. S3StorageProperties -> ObjectStoreInfoPB